janusgraph-es/src/main/java/org/janusgraph/diskstorage/es/ElasticSearchIndex.java

```diff
@@ -245,6 +245,10 @@
                 "this parameter to true but we do recommend to reindex your indexes and don't use this parameter.",
             ConfigOption.Type.MASKABLE, false);
 
+    public static final ConfigOption<Integer> RETRY_ON_CONFLICT =
+        new ConfigOption<>(ELASTICSEARCH_NS, "retry_on_conflict",
+            "Specify how many times should the operation be retried when a conflict occurs. Default: 0.", ConfigOption.Type.MASKABLE, 0);
```

Comment:
As I understand if the user don't specift this parameter then the default value will not be used. So, I would remove `Default: 0` from documentation.

Reply:
removed "Default: 0"